src/project-css/app/Http/Controllers/TableController.php

```diff
@@ -538,6 +635,12 @@ public function worker(Request $request) {
 
       // Counter for processed
       $prcssd = 0;
+      // Counter for failed
+      //$failed = 0;
+
+      // set last ErrRow to null
+      $lastErrRow = NULL;
+      $savedTkns = NULL;
 
       // For each line
       while ($curFltFleObj->valid()) {
```

Comment:
Lots of nested logic here, why so much?  What is the end goal and how can the goal of each method be more simplified.